pandas/tests/indexing/test_loc.py

```diff
@@ -1552,6 +1552,40 @@ def test_loc_getitem_str_timedeltaindex(self):
         sliced = df.loc["0 days"]
         tm.assert_series_equal(sliced, expected)
 
+    @pytest.mark.parametrize("indexer_end", [None, "2020-01-02 23:59:59.999999999"])
```

Comment:
do we need/want tzaware case?

Reply:
I don't think we need them, since we are indexing with strings?

Reply:
i meant for the index to be tzaware

Reply:
Ah, thx. Is this the right fixture?